CompressImagesFunction/CompressImages.cs

```diff
@@ -119,6 +117,21 @@ public static bool Run(CompressimagesParameters parameters, ILogger logger)
             var branchAgain = Commands.Checkout(repo, KnownGitHubs.BranchName);
             repo.Reset(ResetMode.Hard, commitToKeep.Sha);
 
+            // verify images are not corrupted by reading from git
+            // see https://github.com/dabutvin/ImgBot/issues/273
+            try
+            {
+                foreach (var image in optimizedImages)
+                {
+                    new MagickImage(image.OriginalPath).Dispose();
+                }
+            }
+            catch (MagickCorruptImageErrorException)
```

Comment:
You might want to catch `MagickErrorException` instead to make sure you catch all errors.